src/features/user-dashboard/components/UserDetailForm.tsx

```diff
@@ -115,7 +128,7 @@ export const UserDetailForm: FC<Props> = ({
           />
           <Form.Control.Feedback type='invalid'>{errors.role?.message}</Form.Control.Feedback>
           <Form.Text className='text-muted'>
-            For more information about the different roles and what they can do, see our <a href='#'>FAQ</a>
+            <Trans i18nKey="userDetail.roleInfo">For more info regarding roles...</Trans>
```

Comment:
Full string moved into translation files, what's shown here is just placeholder and providing context for devs.